project1/models.py

```diff
@@ -24,4 +24,7 @@ def __init__(self, username, email, password, created) :
         self.username = username
         self.email = email
         self.password = password
-        self.created = created
\ No newline at end of file
+        self.created = created
```

Comment:
How about setting the value for created with the datatime.now() in the constructor?

Reply:
Yes, we can do that though. but, what makes the difference in doing that in the application.py

Reply:
It is a way to protect it from inconsistencies. OOP advantage

Reply:
Okay.